### ISSUE TITLE
Broken dependency

### ISSUE THREAD

**Issue by ulfryk** [yannickcr/gulp-jscpd#5]
Hi there ;)

in `package.json` there is:

```
    "jscpd": "^0.4.2",
```

it would be better if it was:

```
    "jscpd": "0.4.2",
```

or even:

```
    "jscpd": "0.4.3",
```

Now, because of `^` npm downloads version "0.5.0" which has different api so 'gulp-jscpd' throws while executed:

```
module.js:338
    throw err;
    ^

Error: Cannot find module 'jscpd/src/tokenizer/TokenizerFactory'
  at Function.Module._resolveFilename (module.js:336:15)
  at Function.Module._load (module.js:286:25)
  at Module.require (module.js:365:17)
  at require (module.js:384:17)
  at Object.<anonymous> (/Users/jstrojewski/Projects/swingdev/globality-client/node_modules/gulp-jscpd/lib/gulp-jscpd.js:10:24)
  at Module._compile (module.js:434:26)
  at Object.Module._extensions..js (module.js:452:10)
  at Module.load (module.js:355:32)
  at Function.Module._load (module.js:310:12)
  at Module.require (module.js:365:17)
  at require (module.js:384:17)
  at Object.<anonymous> (/Users/jstrojewski/Projects/swingdev/globality-client/gulp/jscpd.js:4:13)
  at Module._compile (module.js:434:26)
  at Object.Module._extensions..js (module.js:452:10)
  at Module.load (module.js:355:32)
  at Function.Module._load (module.js:310:12)
  at Module.require (module.js:365:17)
  at require (module.js:384:17)
  at requireDir (/Users/jstrojewski/Projects/swingdev/globality-client/node_modules/require-dir/index.js:116:33)
  at Object.<anonymous> (/Users/jstrojewski/Projects/swingdev/globality-client/Gulpfile.js:6:23)
  at Module._compile (module.js:434:26)
  at Object.Module._extensions..js (module.js:452:10)
  at Module.load (module.js:355:32)
  at Function.Module._load (module.js:310:12)
  at Module.require (module.js:365:17)
  at require (module.js:384:17)
  at Liftoff.handleArguments (/Users/jstrojewski/local/lib/node_modules/gulp/bin/gulp.js:116:3)
  at Liftoff.<anonymous> (/Users/jstrojewski/local/lib/node_modules/gulp/node_modules/liftoff/index.js:192:16)
  at module.exports (/Users/jstrojewski/local/lib/node_modules/gulp/node_modules/liftoff/node_modules/flagged-respawn/index.js:17:3)
  at Liftoff.<anonymous> (/Users/jstrojewski/local/lib/node_modules/gulp/node_modules/liftoff/index.js:185:9)
  at /Users/jstrojewski/local/lib/node_modules/gulp/node_modules/liftoff/index.js:159:9
  at /Users/jstrojewski/local/lib/node_modules/gulp/node_modules/v8flags/index.js:79:12
  at /Users/jstrojewski/local/lib/node_modules/gulp/node_modules/v8flags/index.js:35:7
  at doNTCallback0 (node.js:407:9)
  at process._tickCallback (node.js:336:13)
  at Function.Module.runMain (module.js:477:11)
  at startup (node.js:117:18)
  at node.js:951:3
```

So please change `package.json` not to use `^` or update library so it works with jscpd 0.5.0

**Comment by yannickcr** [comment]
As a quick fix I published `0.0.6` with jscpd fixed at `0.4.2`.

I will publish an updated version for jscpd `0.5.0` in the next days.

**Comment by ulfryk** [comment]
Thanks :)
